scripts/data_contamination/run_contamination_beam.py

```diff
@@ -0,0 +1,86 @@
+import json
+import apache_beam as beam
+import argparse
+
+from typing import Callable, List
+
+from contamination_beam import ComputeAndWriteContaminationStats
+
+
+def get_extract_text_function(input_format: str):
+    def extract_text_from_the_pile_document(document: str) -> str:
+        return json.loads(document)["text"]
+
+    def extract_text_from_raw_document(document: str) -> str:
+        return document.rstrip("\n")
+
+    if input_format == "raw":
+        return extract_text_from_raw_document
+    elif input_format == "the_pile":
+        return extract_text_from_the_pile_document
+    else:
+        raise NotImplementedError(f"Unknown input format {input_format}")
+
+
+def main():
+    parser = argparse.ArgumentParser()
```

Comment:
Fair bit of duplicate code here, could share and reduce waste, but ultimately each partner will likely only want one script rather than two ways of running it

Reply:
I think this is fine.